Templates/Resources/Templates/___FILEBASENAME___Interfaces.swift.erb

```diff
@@ -9,22 +9,28 @@
 //
 
 import UIKit
```

Comment:
Should we replace UIKit with SwiftUI here for `@template.generate_swift_ui_view`?